.github/workflows/pull_request_tests.yaml

```diff
@@ -0,0 +1,40 @@
+name: Test Action in CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  CLANG_TIDY_VERSION: 17.0.1
+  CMAKE_VERSION: 3.21.3
+  # cannot use > 3.11 due to https://github.com/ssciwr/clang-tidy-wheel/issues/66
+  PYTHON_VERSION: 3.11
+
+jobs:
+  check_action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "${{ env.PYTHON_VERSION }}"
+      - name: Install dependencies
+        run: |
+          pip install clang-tidy==${CLANG_TIDY_VERSION} cmake==${CMAKE_VERSION}
+      - name: Configure Test project
+        run: |
+          mkdir build
+          cd build
+          cmake ../tests
+      - name: Run clang-tidy
+        run: |
+          cd build
+          cmake --build . --target clang-tidy
+      - name: Test Suggest Fixes
+        uses: ./
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          request_change: false
```

Comment:
```suggestion
          request_changes: false
```

Reply:
Thanks. I so wish we could make github actions promote the resulting warnings from the typo to errors.